\documentclass[10pt]{article}

\usepackage{calc}
\usepackage{paralist} % This gives us fun enumeration environments. compactitem will be nice.
\usepackage{enumitem}

\reversemarginpar % Layout: Puts the section titles on left side of page

%% Use these lines for letter-sized paper
\usepackage[paper=letterpaper,
            %includefoot, % Uncomment to put page number above margin
            marginparwidth=1.0in,     % Length of section titles
            marginparsep=.05in,       % Space between titles and text
            margin=.70in,               % 1 inch margins
            includemp]{geometry}

\setlength{\parindent}{0in} %% More layout: Get rid of indenting throughout entire document


\usepackage{fancyhdr,lastpage}
\pagestyle{fancy}
%\pagestyle{empty}      % Uncomment this to get rid of page numbers
\fancyhf{}\renewcommand{\headrulewidth}{0pt}
\fancyfootoffset{\marginparsep+\marginparwidth}
\newlength{\footpageshift}
\setlength{\footpageshift}
          {0.5\textwidth+0.5\marginparsep+0.5\marginparwidth-2in}
\lfoot{\hspace{\footpageshift}%
       \parbox{4in}{\, \hfill %
%                    \arabic{page}  of \protect\pageref*{LastPage} % +LP
                    \arabic{page}                               % -LP
                    \hfill \,}}

% Finally, give us PDF bookmarks
\usepackage{color,hyperref}
\definecolor{darkblue}{rgb}{0.0,0.0,0.3}
\hypersetup{colorlinks,breaklinks,
            linkcolor=darkblue,urlcolor=darkblue,
            anchorcolor=darkblue,citecolor=darkblue}


%%%%%%%%%%%%%%%%%%%%%%%%%%% Helper Commands %%%%%%%%%%%%%%%%%%%%%%%%%%%%

% The title (name) with a horizontal rule under it. e.g. Usage: \makeheading{name}
\newcommand{\makeheading}[1]%
        {\hspace*{-\marginparsep minus \marginparwidth}%
         \begin{minipage}[t]{\textwidth+\marginparwidth+\marginparsep}%
                {\large \bfseries #1}\\[-0.15\baselineskip]%
                 \rule{\columnwidth}{1pt}%
         \end{minipage}}

% The section headings. e.g. Usage: \section{section name}
\renewcommand{\section}[2]%
        {\pagebreak[2]\vspace{1.3\baselineskip}%
         \phantomsection\addcontentsline{toc}{section}{#1}%
         \hspace{0in}%
         \marginpar{
         \raggedright \scshape #1}#2}

% An itemize-style list with lots of space between items
\newenvironment{outerlist}[1][\enskip\textbullet]%
        {\begin{itemize}[#1]}{\end{itemize}%
         \vspace{-.6\baselineskip}}

% An environment IDENTICAL to outerlist that has better pre-list spacing
% when used as the first thing in a \section
\newenvironment{lonelist}[1][\enskip\textbullet]%
        {\vspace{-\baselineskip}\begin{list}{#1}{%
        \setlength{\partopsep}{0pt}%
        \setlength{\topsep}{0pt}}}
        {\end{list}\vspace{-.6\baselineskip}}

% An itemize-style list with little space between items
\newenvironment{innerlist}[1][\enskip\textbullet]%
        {\begin{compactitem}[#1]}{\end{compactitem}}

% An environment IDENTICAL to innerlist that has better pre-list spacing
% when used as the first thing in a \section
\newenvironment{loneinnerlist}[1][\enskip\textbullet]%
        {\vspace{-\baselineskip}\begin{compactitem}[#1]}
        {\end{compactitem}\vspace{-.6\baselineskip}}

% To add some paragraph space between lines.
% This also tells LaTeX to preferably break a page on one of these gaps
% if there is a needed pagebreak nearby.
\newcommand{\blankline}{\quad\pagebreak[2]}

% Uses hyperref to link DOI
\newcommand\doilink[1]{\href{http://dx.doi.org/#1}{#1}}
\newcommand\doi[1]{doi:\doilink{#1}}


%%%%%%%%%%%%%%%%%%%%%%%% End Helper Commands %%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%% Begin CV Document %%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}
\makeheading{Stephanie C. Hicks \hfill Updated \today}

\section{Contact Information}
%
% NOTE: Mind where the & separators and \\ breaks are in the following
%       table.
%
% ALSO: \rcollength is the width of the right column of the table
%       (adjust it to your liking; default is 1.85in).
%
\newlength{\rcollength}\setlength{\rcollength}{2.05in}%
%
\begin{tabular}[t]{@{}p{\textwidth-\rcollength}p{\rcollength}}
\href{http://bcb.dfci.harvard.edu}{Department of Biostatistics and Computational Biology} & \\
\href{http://www.dana-farber.org}{Dana-Farber Cancer Institute} & twitter: @stephaniehicks \\ 
450 Brookline Avenue   & email: \href{mailto:shicks@jimmy.harvard.edu}{shicks@jimmy.harvard.edu} \\ %Fax: 713.348.5476  \\
Boston, MA 02215 USA   & website: \href{http://www.stephaniehicks.com}{stephaniehicks.com} 
\end{tabular}

%\href{http://www.hsph.harvard.edu}{Harvard School of Public Health} 

%\section{Objective}
%%
%Placement in a post-doctoral or academic faculty research position
%Research and develop statistical and population genetics tools in application for high-throughput sequencing technologies%to better understand genetic variation within and among populations and its association with phenotypic variation
% Placement in a post-doctoral or academic faculty research position researching and developing statistical and population genetics tools in application for high-throughput sequencing technologies

%\section{Objective} 
%\begin{bibsection}
%\item Placement in an interdisciplinary post-doctoral position researching and developing statistical and bioinformatics methodologies and tools for the analysis of high-throughput genomic data
%\end{bibsection}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%% Research Interests %%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Research Interests}
My research interests focus around developing statistical methods and tools in application for genomics data. Currently, I am focused on methods for pre-processing, normalizing and analyzing gene expression and DNA methylation data based on microarray and next-generation sequencing platform technologies.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%% Current Appointment %%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\section{Current Appointment}
\textbf{Postdoctoral Research Fellow}, Boston MA  \hfill 2013 - Present
\begin{innerlist}
\item[] Department of Biostatistics and Computational Biology, Dana-Farber Cancer Institute
\item[] Department of Biostatistics, Harvard T.H. Chan School of Public Health
\item[] \textit{Mentor: Rafael Irizarry} 
\end{innerlist}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%% Eduction %%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Education}
\vskip -.25in
\begin{table}[h]
\begin{tabular}{cl}
2007 - 2013 & \textbf{PhD. $\&$ M.A., Statistics}, Rice University, Houston, TX USA \\
& \textit{Thesis Advisors: Marek Kimmel, Ph.D. (Rice, Statistics) and } \\
& \hspace{0.95in} \textit{  Sharon Plon, M.D., Ph.D. (Baylor College of Medicine) } \\
& \textit{Thesis Title: Probabilistic Models for Genetic and Genomic Data with Missing Information} \\
& \\
2003 - 2007 & \textbf{B.S., Mathematics}, Louisiana State University, Baton Rouge, LA USA \\
&  \textit{magna cum laude}, Phi Beta Kappa 
\end{tabular}
\end{table}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%% Professional Experience %%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Professional Experience}
\vskip -.25in
\begin{table}[h]
\begin{tabular}{ll}
2013 -  & \textbf{Postdoctoral Research Fellow}, DFCI and HSPH, Boston, MA   \\
& \textit{Mentor: Rafael Irizarry} \\
& \\
2007 - 2013 &  \textbf{Graduate Student Researcher}, Rice University, Houston, TX \\
& \textit{Advisors: Marek Kimmel and Sharon Plon (Baylor College of Medicine)} \\
& \\
2006 &  \textbf{Undergraduate Researcher (REU)}, Statistics, University of Wisconsin, Madison, WI \\
& \textit{Mentors: Murray  Clayton and Jo Handelsman} \\
& \\
2005 &  \textbf{Undergraduate Researcher (REU)}, Statistics,  Rice University, Houston, TX  \\
& \textit{Mentor: Javier Rojo} \\
& \\
2005 - 2007 &  \textbf{Undergraduate Researcher}, Psychology, LSU, Baton Rouge, LA  \\
& \textit{Mentor: Robert Mathews} \\
& \\
2005 - 2007 &  \textbf{Undergraduate Tutor}, Roadmap 2 Redesign Program, LSU, Baton Rouge, LA \\
& \textit{ Tutored college algebra, trigonometry and pre-calculus students.} \\
& \\
2003 - 2004 &  \textbf{Undergraduate Tutor}, LSU, Baton Rouge, LA \\
& \textit{ Tutored students at Scotlandville Magnet High School in algebra for 3hrs/week. } \\
\end{tabular}
\end{table}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%% Publications %%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Research}  
\textbf{Pre-prints }
\begin{enumerate}[label= {[\arabic*]}]
\item {\bf Hicks SC}, Irizarry RA. (2014). When to use Quantile Normalization? {\it bioRxiv} \\
doi: http://dx.doi.org/10.1101/012203. 
\end{enumerate}


\textbf{Journal Articles}
\begin{enumerate}[label= {[\arabic*]}]
\item Neskey DM, Osman AA, Ow TJ, Katsonis P, McDonald T, {\bf Hicks SC}, Hsu TK, Pickering CR, Ward A, Patel A, Yordy JS, Skinner HD, Giri U, Sano D, Story MD, Beadle BM, El-Naggar AK, Kies MS, William WN, Caulin C, Frederick M, Kimmel M, Myers JN, Lichtarge O. (2015). Evolutionary Action score of TP53 (EAp53) identifies high risk mutations associated with decreased survival and increased distant metastases in head and neck cancer. {\it Cancer Res} [Epub ahead of print]. PMID: 25634208
\item Berger RL, Li LT, {\bf Hicks SC}, Liang MK. (2014). Suture versus preperitoneal polypropylene mesh for elective umbilical hernia repairs. {\it J Surg Res} {\bf 192}: 426-431. PMID: 24980854 
\item Brahmbhatt R, Carter SA, {\bf Hicks SC}, Berger DH, Liang MK. (2014). Identifying Risk Factors for Surgical Site Complications after Laparoscopic Ventral Hernia Repair: Evaluation of the Ventral Hernia Working Group Grading System. {\it Surg Infect (Larchmt)} {\bf 15}: 187-193. PMID: 24773169
\item Carter SA, {\bf Hicks SC}, Brahmbhatt R, Liang MK. (2014). Recurrence and Pseudorecurrence after Laparoscopic Ventral Hernia Repair: Predictors and Patient-focused Outcomes. {\it Am Surg} {\bf 80}: 138-148. PMID: 24480213
\item {\bf Hicks SC}. (2014). When Women in Statistics Come to Know Their Power. {\it Chance} {\bf 27}. 
\item Li LT, {\bf Hicks SC}, Davila JA, Kao LS, Berger RL, Arita NA, Liang MK. (2014). Circular Closure is Associated with the Lowest Rate of Surgical Site Infection Following Stoma Reversal:A Systematic Review and Multiple Treatment Meta-analysis. {\it Colorectal Dis} {\bf 16}: 406-416. PMID: 24422861
\item Li LT, Brahmbhatt R, {\bf Hicks SC}, Davila JA, Berger DH, Liang MK. (2014). Prevalence of Surgical Site Infection at the Stoma Site following Four Skin Closure Techniques: A Retrospective Cohort Study. {\it Dig Surg} {\bf 31}: 73-78. PMID: 24776653
\item Li LT, Jafrani RJ, Becker NS, Berger RL,  {\bf Hicks SC}, Davila JA, Liang MK. (2014). Outcomes of acute versus elective primary ventral hernia repair. {\it J Trauma Acute Care Surg} {\bf 76}: 523-528. PMID: 24458061
\item Liang MK, Li LT, Nguyen MT, Berger RL, {\bf Hicks SC}, Kao LS. (2014). Abdominal reoperation and mesh explantation following open ventral hernia repair with mesh. {\it Am J Surg} {\bf 204}: 670-676. PMID: 25241955. 
\item Liang MK, Berger RL, Nguyen MT, {\bf Hicks SC}, Li LT, Leong M. (2014). Outcomes with Porcine Acellular Dermal Matrix versus Synthetic Mesh and Suture in Complicated Open Ventral Hernia Repair. {\it Surg Infect} {\bf 15}: 506-512. PMID: 25215466. 
\item Nguyen MT, Berger RL, {\bf Hicks SC}, Davila JA, Li LT, Kao LS, Liang MK. (2014). Comparison of Outcomes of Synthetic Mesh vs Suture Repair of Elective Primary Ventral Herniorrhaphy: A Systematic Review and Meta-analysis. {\it JAMA Surg} {\bf 31}: 73-78. PMID: 24554114 
\item Nguyen MT, Phatak UR, Li LT, {\bf Hicks SC}, Moffett JM, Arita NA, Berger RL, Kao LS, Liang MK. (2014). Review of stoma site and midline incisional hernias after stoma reversal. {\it J Surg Res} {\bf 190}: 504-509. PMID: 24560428 
\item Berger RL, Li LT, {\bf Hicks SC}, Davila JA, Kao LS, Liang MK. (2013). Development and validation of a risk-stratification score for surgical site occurrence and surgical site infection after open ventral hernia repair. {\it J Am Coll Surg.} {\bf 217}: 974-982. PMID: 24051068
\item Clapp ML, \textbf{Hicks SC}, Awad SS, Liang MK. (2013). Trans-cutaneous Closure of Central Defects (TCCD) in Laparoscopic Ventral Hernia Repairs (LVHR). {\it World J Surg}. {\bf 37}: 42-51. PMID: 23052806.
\item Liang MK, Clapp M, Li LT, Berger RL, {\bf Hicks SC}, Awad S. (2013). Patient Satisfaction, Chronic Pain, and Functional Status following Laparoscopic Ventral Hernia Repair. {\it World J Surg}. {\bf 37}: 530-537. PMID: 23212794. 
\item Liang MK, Li LT, Avellaneda A, Moffett JM, {\bf Hicks SC}, Awad SS. (2013). Outcomes and Predictors of Incisional Surgical Site Infection in Stoma Reversal. {\it JAMA Surg} {\bf 148}: 183-189. PMID: 23426597. 
\item Liang MK, Berger RL, Li LT, Davila JA, {\bf Hicks SC}, Kao LS. (2013). Outcomes of laparoscopic vs open repair of primary ventral hernias. {\it JAMA Surg} {\bf 148}: 1043-1048. PMID: 24005537 
\item Neskey DM, Klein JD, {\bf Hicks S}, Garden AS, Bell DM, El-Naggar Ak, Kies MS, Weber RS, Kupferman ME. (2013). Prognostic Factors Associated With Decreased Survival in Patients With Acinic Cell Carcinoma. {\it JAMA Otolaryngol Head Neck Surg} {\bf 139}: 1195-1202. PMID: 24076756
\item Nguyen MT, Li LT, {\bf Hicks SC}, Davila JA, Suliburk JW, Leong M, Kao LS, Berger DH, Liang MK. (2013). Readmission following open ventral hernia repair: incidence, indications, and predictors. {\it Am J Surg} {\bf 206}: 942-948. PMID: 24296099
\item Subramanian A, Clapp ML, {\bf Hicks SC}, Awad SS, Liang MK. (2013). Laparoscopic ventral hernia repair: Primary versus secondary hernias. {\it J Surg Res} {\bf 181}: e1-5. PMID: 22795342.
\item \textbf{Hicks S}, Plon SE, Kimmel M. (2013). Statistical Analysis of Missense Mutation Classifiers. {\it Hum Mut} {\bf 34}: 405-406. PMID: 23086893. 
\item Cheung HC\footnote{ `Highly Accessed' on BioMed Central}, San Lucas FA, \textbf{Hicks S}, Chang K, Bertuch AA, Ribes-Zamora A. (2012). An S/T-Q cluster domain census unveils new putative targets under Tel1/Mec1 control. {\it BMC Genomics} {\bf 13}: 664. PMID: 23176708. 
\item \textbf{Hicks S}\footnote{Cited and Discussed in Nature {\bf 482}: 257-262. 09 Feb 2012. PMID: 22318607}, Wheeler DE, Plon SE, Kimmel M. (2011). Prediction of Missense Mutation Functionality Depends on both the Algorithm and Sequence Alignment Employed. {\it Hum Mut}. {\bf 32}: 661-668. PMID: 21480434.
\end{enumerate}



\vskip .2in
\textbf{Invited Talks} 
\begin{enumerate}[label= {[\arabic*]}]
	\item Why Statistics Matters in the Analysis of Genomics Data. Computational Biology Seminar, Department of Biology, Louisiana State University. 2015 Feb 11. Baton Rouge, LA. 
	\item Normalization of DNA methylation and Gene Expression Data in the Context of Global Variation.  Bioinformatics Meeting, Division of Immunology, Harvard Medical School. 2014 Sept 18. Boston, MA. 
\end{enumerate}


\vskip .2in
\textbf{Contributed Talks} 
\begin{enumerate}[label= {[\arabic*]}]
	\item {\bf Hicks SC}, Irizarry R. quantro: When should you use quantile normalization? Flashlight talk at \emph{Bioconductor Conference}. 2014 July 30-August 1. Boston, MA. 
	\item {\bf Hicks S}, Plon SE, Kimmel M. Modeling Discovery Of Functional SNPs From Genome Scale Data. \emph{Joint Statistical Meetings}. 2011 Aug 5. Miami, FL. 
	\item {\bf Hicks S}, Plon SE, Wheeler DE, Kimmel M. Prediction of Missense Mutation Functionality Depends on both the Algorithm and Sequence Alignment Employed. \emph{Human Genome Variation Society's Exploring the Functional Consequences of Genomic Variation Meeting}. 2010 Nov 1. Washington, D.C.
\end{enumerate}


\vskip .2in
\textbf{Conference Proceedings} 
\begin{enumerate}[label= {[\arabic*]}]
	\item Saliba J, Trevino LR, Meng Q, Zabriskie R, Powell B, {\bf Hicks S}, Kimmel M, Cheung H, Muzny DM, Reid JG, Wheeler D, Gibbs RA, and Plon SE.  Abstract 5113: Functional analysis of genomic variants identified through whole exome sequencing for susceptibility to lymphocytic leukemia. {\it AACR 103rd Annual Meeting}. 2012 Mar 31-Apr 4. Chicago, IL. 
	\item Saliba J, Zabriskie R, Powell B, {\bf Hicks S}, Kimmel M, Cheung H, Ritter D, Muzny DM, Reid JG, Wheeler DA, Gibbs RA, Plon SE. Abstract A8: Functional analysis of genomic variants identified through whole exome sequencing of pediatric lymphocytic leukemia kindreds. {\it AACR Special Conference: Pediatric Cancer at the Crossroads: Translating Discovery into Improved Outcomes}. 2013 Nov 3-6. San Diego, CA. 

\end{enumerate}


\vskip .2in

\textbf{Selected Poster Presentations} 
\begin{enumerate}[label= {[\arabic*]}]
\item {\bf Hicks S}, Irizarry R. When to use Quantile Normalization? Poster presented at: \emph{2014 Women in Statistics Conference}. 2014 May 15-17. Cary, NC.
\item {\bf Hicks S}, Irizarry R. Estimating cell composition of whole blood using differentially methylated regions. Poster presented at: \emph{2013 PQG Conference}. 2013. Nov 14-15. Boston, MA. 
\item {\bf Hicks S}, Plon SE, Kimmel M. postMUT: A Statistical Tool for Combining Predictions of Missense Mutation Functionality using Capture-Recapture Methods. Poster presented at: \emph{63nd Annual Meeting of The American Society of Human Genetics}. 2013. Oct 22-26. Boston, MA.
\item Kimmel M, {\bf Hicks S}, Plon SE. Applications of Hidden Markov Models with Conditional Emission Probabilities to Identify Regions of Identity-By-Descent in Whole-Exome Sequencing Data. Poster presented at: \emph{63nd Annual Meeting of The American Society of Human Genetics}. 2013. Oct 22-26. Boston, MA.
\item {\bf Hicks S}, Plon SE, Kimmel M. Capture-Recapture Models for Evaluation of Algorithms Estimating Functionality of Missense Mutations. Poster presented at: \emph{62nd Annual Meeting of The American Society of Human Genetics}. 2012 Nov 6-12. San Francisco, CA. %(Abstract $\#$ 3549F)
\item {\bf Hicks S}, Plon SE, Kimmel M. Bernoulli mixture models in application of the evaluation of algorithms estimating functionality of missense mutations. Poster presented at: \emph{Beyond The Genome}. 2012 Sept 27-29. Boston, MA.
\item {\bf Hicks S}, Plon SE, Kimmel M. Bernoulli Mixture Models in Application of the Evaulation of Algorithms Estimating Functionality of Missense Mutations. Poster presented at: \emph{International Conference in Stochastic Processes}. 2012. Aug 21-25. Houston, TX. 
\item {\bf Hicks S}, Plon SE, Kimmel M. Using a Second-order Hidden Markov Model to Identify Regions of Identity-By-Descent in Exome Sequencing Data. Poster presented at: \emph{12th International Congress of Human Genetics/61st Annual Meeting of The American Society of Human Genetics}. 2011 Nov 12-15. Montreal, Canada.
\item {\bf Hicks S}, Plon SE, Wheeler DE, Kimmel, M. Prediction of Missense Mutation Functionality Depends on both the Algorithm and Sequence Alignment Employed. Poster presented at: \emph{60th Annual Meeting of the American Society of Human Genetics}. 2010 Nov 2-6. Washington, D.C.
\item {\bf Hicks S}, Plon SE, Wheeler DE, Kimmel, M. Prediction of Missense Mutation Functionality Depends on both the Algorithm and Sequence Alignment Employed. Poster presented at: \emph{4th Annual Meeting of the Genomics of Common Diseases}. 2010 Oct 6-9. Houston, TX.
\item Cheung HC, San Lucas F, {\bf Hicks S}, Chang K, Plon SE, Bertuch AA, Ribes-Zamora A. S/T-Q Cluster Domains in Saccharomyces cerevisiae: a bioinformatic census and analysis. Poster presented at: \emph{BioScience Research Collaborative Grand Opening}. 2010 Apr 1. Houston, TX. 
\item {\bf Hicks S}, Plon SE, Wheeler DE, Kimmel, M. Predicting Functionality of Missense Mutations with Varying Levels of Evolutionary Depth. Poster presented at: \emph{Computational \& Theoretical Biology Symposium}. 2009 Dec 4-6. Houston, TX.  
\item {\bf Hicks S}, Plon SE, Wheeler DE, Kimmel, M. Predicting Functionality of Missense Mutations with Varying Levels of Evolutionary Depth. Poster presented at: \emph{59th Annual Meeting of the American Society of Human Genetics}. 2009 Oct 20-24. Honolulu, HI.
\end{enumerate}

\vskip .2in

\textbf{Ph.D. Dissertation}
\begin{enumerate}[label= {[\arabic*]}]
\item {\bf Hicks SC}. Probabilistic Models for Genetic and Genomic Data with Missing Information, Ph.D. Thesis, Rice University, 2013. 
\end{enumerate}

%\vskip .2in



   
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%% Software %%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Software}
\textbf{Bioconductor and Github}
\begin{enumerate}[label= {[\arabic*]}]
\item \textit{quantro}: An R-package that can be used to test for differences between groups of distributions to guide the choice if quantile normalization should be used [\href{http://www.bioconductor.org/packages/release/bioc/html/quantro.html}{Available on Bioconductor}]
\item \textit{quantroSim}: A supporting data simulation R-package for the \textit{quantro} R-package 
to simulate gene expression and DNA methylation data  [\href{https://github.com/stephaniehicks/quantroSim}{Available on Github}] 
\item \textit{postMUT}: A tool implemented in Perl and R to predict the functionality of missense mutations [\href{https://github.com/stephaniehicks/postMUT}{Available on Github}]
\end{enumerate}     
  

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%% Teaching Experience %%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\vskip .1in

\section{Teaching Experience}
%\vskip -.30in
\begin{table}[h]
\begin{tabular}{ll}
2014 & \textbf{Lead Teaching Fellow}, \href{http://cs109.github.io/2014/}{Data Science (Harvard - CS109)} \\
& \textit{Held weekly lab sections. Developed course material, homework and solutions. } \\
& \textit{Coordinated and directed 25 teaching fellows. All material available on}:  \href{https://github.com/cs109/2014}{Github}\\
& \textit{Course taught by Rafael Irizarry and Verena Kaynig-Fittkau.} \\
& \\
2014 & \textbf{Discussion Leader}, \href{https://www.edx.org/course/harvardx/harvardx-ph525x-data-analysis-genomics-1401}{Data Analysis for Genomics (HarvardX - PH525x)} \\
& \textit{Course taught by Rafael Irizarry with Teaching Assistant Mike Love.} \\
& \\
2010, 2011 & \textbf{Teaching Assistant}, Probability in Bioinformatics and Genetics (Rice - STAT 423/623)   \\
& \textit{Grade homeworks, provide solutions, hold office hours and gave several guest lectures. } \\
& \\
2010 & \textbf{Teaching Assistant}, Applied Stochastic Processes (Rice - STAT 552)   \\
& \textit{Grade homeworks, provide solutions, hold office hours and gave several guest lectures. } \\
& \\
2008, 2009 & \textbf{Lab Instructor}, Introduction to Statistics for the Biosciences (Rice - STAT 305)   \\
 & \textit{Hold labs 3hr/week to teach students to use R.} \\
& \\
2007, 2008 & \textbf{Teaching Assistant}, Probability and Statistics (Rice - STAT 310)   \\
& \textit{Grade homeworks, provide solutions, hold office hours and gave several guest lectures. } \\
& \\
\end{tabular}
\end{table}

%Rice University \hfill 2007 - 2013 
%\begin{innerlist}
%        \item[] Teaching Assistant for Stat 423/623: Probability in Bioinformatics and Genetics 
%        \item[] Teaching Assistant for Stat 552: Applied Stochastic Processes
%        \item[] Teaching Assistant for Stat 310: Probability and Statistics
%        \item[] Lab Instructor for Stat 305: Introduction to Statistics for the Biosciences
%
%\end{innerlist}
%
%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%% Awards %%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\section{Grants and Awards}
\vskip -0.25in
\begin{table}[h]
\begin{tabular}{ll}
2014 & Travel award for the Women in Statistics Conference 2014 \\
2011 & Travel and tuition award for the 16th Annual Summer  Institute in Statistical Genetics \\ 
& at University of Washington (Population Genetics, Coalescent Theory modules) \\
2008 - 2011 & NIH T32 Training Grant Fellow, Rice University \\
2007 & Inducted into Phi Beta Kappa \\
2007 & LSU Austin Chapter Scholarship Award \\
2005 - 2007 & LA-STEM Research Scholars, LSU \\
2004 - 2005 & HMM Professors Program, LSU \\
2003 - 2007 & TOPS Tuition Award, LSU \\
\end{tabular}
\end{table}




%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%% Service %%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
%\section{Service}
%\vskip -.35in
%\begin{table}[h]
%\begin{tabular}{ll}
%2008 - 2012 & \textbf{Coordinator of the Computational Biology Seminar}, Rice University \\
%& \textit{Responsible for maintaining the website, scheduling and announcing the speakers.} \\
%& \\
%2008 - 2009 & \textbf{VP of Graduate Mathematics, Applied and Computational } \\
%& \hspace{.2in} \textbf{ Mathematicians and Statisticians}, Rice University \\
%& \textit{To foster an environment of growth and enthusiasm for math among the students }\\
%& \hspace{.2in} \textit{ through intellectual and social events.} \\
%& \\
%2005 - 2007 & \textbf{Treasurer of LSU Habitat for Humanity} \\
%\end{tabular}
%\end{table}
%
   

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%% Technical Skills %%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Technical Skills}
\textbf{Programming}: R, Python, Git, Perl, LaTeX{}, SAS, Matlab, Shell scripting \\
\textbf{Operating Systems}: Mac OS X, Unix, Windows
   

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%% Reviewed journal articles %%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Journal Referee / Reviewer}
\emph{Bioinformatics, Biometrics, BMC Medical Genetics, Human Mutation}

\vskip .2in
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%% Statistical Expertise %%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%\section{Statistical Expertise}
%%
%Statistical Learning, Statistical Genetics, Stochastic Processes,  Survival Analysis, Bayesian Data Analysis, Multivariate Analysis, Biostatistics, Probabilistic Models in Bioinformatics, General Linear Models, Statistical Sampling
%


        
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%% Selected Memberships %%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
   
\section{Professional Memberships}
%
Member, American Statistical Association (ASA) \\
Member, American Mathematical Society (AMS) \\
Member, American Society of Human Genetics (ASHG)




%
%\section{References}
%%
% Marek Kimmel, Ph.D. \\
%Professor, Department of Statistics \\ 
%Rice University \\
%kimmel@rice.edu, (713) 348-5255 \\
%
%Sharon Plon, M.D., Ph.D. \\
%Professor, Departments of Pediatrics and Human and Molecular Genetics \\
%Human Genome Sequencing Center \\
%Baylor College of Medicine \\
% splon@bcm.edu, (832) 824-4251 \\ 
% 
%Kathy Ensor, Ph.D. \\
%Professor and Chair, Department of Statistics \\
%Rice University \\
%ensor@rice.edu, (713) 348-4687  \\

\end{document}

%%%%%%%%%%%%%%%%%%%%%%%%%% End CV Document %%%%%%%%%%%%%%%%%%%%%%%%%%%%%