---
layout: page
title: Research
---

[CV](CV_StephanieHicks.pdf) &nbsp; | &nbsp; 
[Google Scholar](http://scholar.google.com/citations?user=4T4qnL4AAAAJ) &nbsp; | &nbsp; 
[github](https://github.com/stephaniehicks)  

#### Research Interests
My research interests focus around developing statistical methods and tools in application for genomics and epigenomics data. Currently, I am focused on methods for processing and analyzing DNA methylation and gene expression using microarrays and next-generation sequencing. Here are a few software packages and data packages that I have authored:


#### Software

- [R/quantro](https://github.com/stephaniehicks/quantro): R package to test for global differences between groups of distributions to decide when to use quantile normalization.
- [R/epigenomeSim](https://github.com/stephaniehicks/epigenomeSim): R package package is to simulate data from the epigenome including gene expression and DNA methylation.
- [postMUT](https://github.com/stephaniehicks/postMUT): A tool implemented in Perl and R to predict the functionality of missense mutations.


#### Data Packages

- [myAffyData](https://github.com/stephaniehicks/mycAffyData): AffyBatch object from an experiement using P493-6 cells expressing low or high levels of c-Myc. Data from Loven et al. (2012) Cell 151: 476-482.
- [BackgroundExperimentYeast](https://github.com/stephaniehicks/BackgroundExperimentYeast): AffyBatch object from an experiment to measure NSB and optical noise in yeast.